app/poros/food.rb

```diff
@@ -0,0 +1,17 @@
+class Food
+
+  attr_reader :upc,
+              :description,
+              :brand_owner,
+              :ingredients
+
+  def initialize(data)
+    binding.pry
+
+    @upc = data[:gtinUpc]
```

Comment:
To tighten your logic up even more -- there are some foods where gtinUpc and brandOwner aren't present in their hash, so you could bring this to the next level by adding a nil check to make sure the JSON key is present before accessing its value